### ISSUE TITLE
hyperkit uses excessive CPU when it should be idle

### ISSUE THREAD

**Issue by olix0r** [docker/for-mac#1541]
### Expected behavior

hyperkit doesn't use excessive CPU when it should be totally idle.

### Actual behavior

About once or twice a month, usually after waking my laptop up from sleep, _hyperkit_ consumes one or two full cores worth of CPU. There is literally nothing launched in docker.

Furthermore, when quitting Docker for Mac from the menu, it takes several minutes for hyperkit to stop consuming resources.

### Information

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu

Diagnostic ID: 1D61C9FD-5E70-49D5-A786-95BE18ABFECA

```
Docker for Mac: version: 17.04.0-ce-mac7 (b598153e2)
macOS: version 10.10.5 (build: 14F2315)
logs: /tmp/1D61C9FD-5E70-49D5-A786-95BE18ABFECA/20170412-193456.tar.gz
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     db
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     slirp
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         docker-cli check failed with: (Failure "docker -v: timeout after 10.00s")
[OK]     menubar
[OK]     disk
```

  - A reproducible case if this is a bug, Dockerfiles FTW

Docker for mac is doing literally _nothing_ (as far as I'm concerned) when this is observed.  Just having it running in the background, and this crops up semi-frequently.

### Steps to reproduce the behavior

1. Install Docker for mac
2. Don't run anything in Docker
3. Continue normal usage for some amount of time (in this most recent case: 5 days)

![](https://pbs.twimg.com/media/C62AD3EU4AA-N-T.jpg)
<img width="556" alt="screen shot 2017-04-12 at 7 39 33 pm" src="https://cloud.githubusercontent.com/assets/240738/24988063/5d4ee5b2-1fb8-11e7-88a1-85595c4613b6.png">

**Comment by dsheets** [comment]
This is a known issue with OS X 10.10. We worked around the issue as best as we could by suspending the hypervisor immediately before sleep and resuming it immediately after sleep but this issue still occurs in some cases. We've reported it to Apple and we no longer see reports of the issue with OS X 10.11.6 or macOS 10.12. The best way to fix this issue is to upgrade your OS to 10.11 or 10.12. We have deprecated 10.10 support for this and similar problems.

Thanks for using Docker for Mac!
